README.md

```diff
@@ -116,6 +116,14 @@ and automatically add it to releases list in metainfo. To specify which source
 is the app upstream source, set property `is-main-source` to `true` in the
 checker metadata for that source.
 
+The checkers can also support version constraining, making it possible to e.g.
```

Comment:
Use a sub-section here (with header e.g. `### Version constraining`), then it can be referenced from checkers descriptions like `The [versions](#version-constraining) property is supported`.

Reply:
Fixed.